### ISSUE TITLE
 oid_bbox_trainable_label_map.pbtxt

### ISSUE THREAD

**Issue by username123062** [tensorflow/models#7989]
coco dataset has only 80 object classes, but in the 'oid_bbox_trainable_label_map.pbtxt', it has 545 object classes. So how do you get the 545 object classes? Thanks.

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
What is the top-level directory of the model you are using
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by rkotcherr** [comment]
OID suggests these are the labels for Open Image Dataset...

**Comment by ravikyram** [comment]
@username123062 

It is difficult to understand your issues only based on the information you provided.Please, let us know which pretrained model you are using and share related code .Thanks!

**Comment by ravikyram** [comment]
Automatically closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!
